import java.io.File;
import java.io.IOException;

import qualify.TestCase;

public class Test2 extends MotherTest {

	public Test2() {
		myString = 'hello world';
	}

	public void run() {
		// No exception is thrown during that test execution
		check(b)
		System.out.println('running Test2');
	}

}